cache/cache.go

```diff
@@ -1,37 +0,0 @@
-// Package cache is a generic cache mechanism.
```

Comment:
The description doesn't mention anything about removing cache. What's the reason for removing it?

Reply:
I plan to re-introduce this later, properly isolating it, making it context and more generically metrics and logging-aware.